src/atlas/grid/detail/tiles/LFRicTiles.cc

```diff
@@ -670,6 +670,35 @@ void LFRicCubedSphereTiles::enforceWrapAround(const atlas::idx_t t, atlas::Point
     return;
 }
 
+const PointXY& LFRicCubedSphereTiles::tileCentre(size_t t) const {
+    return tileCentres_[t];
+}
+
+const JacobianXY& LFRicCubedSphereTiles::tileJacobian(size_t t) const {
+    return tileJacobians_[t];
+}
+
+// Centre of each tile in xy space.
+const std::array<PointXY, 6> LFRicCubedSphereTiles::tileCentres_ {
+    PointXY{45., 0.},
+    PointXY{135., 0.},
+    PointXY{225., 0.},
+    PointXY{315., 0.},
+    PointXY{45., 90.},
+    PointXY{45., -90.}
```

Comment:
I think that it would be good if there was a single source of truth for this.

Reply:
Agreed. I'll push a change shortly.

Reply:
`constexpr` only applied to literal types, so I don't think it will work with `PointXY`. I've now changed the `botLeftTile` members to static functions with use the tile centres `+ {+/- 45, +/- 45}`.